src/tools/PrizeTierController/SavePrizeTiersModal/PrizePoolTransactionDisplay.tsx

```diff
@@ -92,7 +92,9 @@ export const PrizePoolTransactionDisplay = (props: {
               onClick={submitPushTransaction}
               state={transaction?.state}
               status={transaction?.status}
-              className={classNames({ 'mb-3': transaction?.state === TransactionState.pending })}
+              className={classNames({
+                'w-full mb-3': transaction?.state === TransactionState.pending
```

Comment:
These styles will only be applied if the transaction is pending. Since we're conditionally hiding it we probably want them applied all the time so you can just set `className='w-full mb-3'`

Reply:
Fixed through acbdfcc